src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 0.3s;
```

Comment:
Please use variable for transition time as you use it in few places
Also please specify which proprty you are transtioning, using transition-property rule or just transition rule with several params, like transition: transform, color $transition time;